ui/components/schema/AddResolverDialog.js

```diff
@@ -34,7 +34,7 @@ class AddResolverDialog extends BaseResolverDialog {
     }
 
     createResolver() {
-        console.log("called create resolver");
+        console.log("called create resolver function");
```

Comment:
Could we not use a logger deb/lib here?

Reply:
How can a function do 2 actions create and resolver? IMHO each function should do one action(encapsulation). 
Then here we will have something as follows. 

```javascript

create() {
   console.log("called create function");
}

resolver() {
   console.log("called resolver function");
}

```

Also, what exactly means resolver? Could we find a better word to clarify what should be done here?